script/main_test.py

```diff
@@ -1,32 +1,37 @@
 import unittest
+import main
 
 
-structure_to_test =[{"phone":"04-56-18-88-34","cell":"06-74-93-14-75"},{"phone":"17533450","cell":"67583818"},{"phone":"075 831 68 55","cell":"075 292 22 27"},{"phone":"(394)-381-6746","cell":"(361)-356-8993"},{"phone":"(00) 1170-0136","cell":"(32) 7367-6536"}]
+class TestRemovesSpecialCharacters(unittest.TestCase):
 
+  def test_remove_special_characters(self):
 
-# class TestRemovesSpecialCharacters(unittest.TestCase):
+    self.assertEqual(main.clear_phone_numbers("04-56-18-88-34"),"0456188834")
+    self.assertEqual(main.clear_phone_numbers("06-74-93-14-75"),"0674931475") 
 
-#   def test_remove_special_characters(self):
-#     dict1 = {
-#       "phone":"04-56-18-88-34",
-#       "cell":"06-74-93-14-75"
-#     }
-#     list_dict = [dict1]
+    self.assertEqual(main.clear_phone_numbers("075 831 68 55"),"0758316855")
+    self.assertEqual(main.clear_phone_numbers("075 292 22 27"),"0752922227")
 
-#     self.assertEquals("04-56-18-88-34", list_dict[0]["phone"])
-#     self.assertEquals("06-74-93-14-75", list_dict[0]["cell"])
+    self.assertEqual(main.clear_phone_numbers("(394)-381-6746"),"3943816746")
+    self.assertEqual(main.clear_phone_numbers("(361)-356-8993"),"3613568993")
 
-#     remove_special_characters_from_phone_numbers(list_dict)
+    self.assertEqual(main.clear_phone_numbers("(00) 1170-0136"),"0011700136")
+    self.assertEqual(main.clear_phone_numbers("(32) 7367-6536"),"3273676536")
 
-#     self.assertEquals("0456188834", list_dict[0]["phone"])
-#     self.assertEquals("0674931475", list_dict[0]["cell"])   
 
-class TestDigitAddition(unittest.TestCase):
-  def test_addition(self):
-    number1 = 2
-    number2 = 4
-    sum_numbers = number1 + number2
-    self.assertTrue(6, sum_numbers)
+  def test_are_no_special_characters(self):
+
+    self.assertEqual(main.clear_phone_numbers("0456188834"),"0456188834")
```

Comment:
Isn't the input and expected data the same here? 
Both are: "0456188834"
What does it test?

Reply:
there are phone numbers in the file that do not contain special characters, I thought to also test them if the function does not destroy something

Reply:
This is very good idea. But 1-2 asserts of this type would be enough